benchmarks/src/main/scala/benchmark/Benchmark.scala

```diff
@@ -151,13 +151,13 @@ abstract class Benchmark(private val conf: BenchmarkConf) {
       queryResults += QueryResult(queryName, iteration, Some(durationMs), errorMsg = None)
       log(s"END took $durationMs ms: $banner")
       log("=" * 80)
-      df
+      r
```

Comment:
nit: Just return df.collect()? In general, single use variables are considered redundant unless they are used to improve readability/line breaks.

Reply:
This is needed here, we're measuring the time it takes to run df.collect() on line 153. df itself is used in two different places.